docs/deployment.rst

```diff
@@ -165,62 +165,35 @@ The timeout for an operation on the Kafka store. This is the maximum time that a
-
-To protect against complete loss of DC A, Kafka cluster A (the source) is replicated to Kafka cluster B (the target). This is achieved by running the :ref:`Replicator` <connect_replicator>` local to the target cluster.
-
-Important Settings
+Complete downtime
 ^^^^^^^^^^^^^^^^^^
 
-``kafkastore.connection.url``
-kafkastore.connection.url should be identical across all schema registry nodes. By sharing this setting, all Schema Registry instances will point to the same ZooKeeper cluster.
-
-``schema.registry.zk.namespace``
-Namespace under which schema registry related metadata is stored in Zookeeper. This setting should be identical across all nodes in the same schema registry.
-
-``master.eligibility``
-A schema registry server with ``master.eligibility`` set to false is guaranteed to remain a slave during any master election. Schema Registry instances in a "slave" data center should have this set to false, and Schema Registry instances local to the shared Kafka cluster should have this set to true.
-
-Setup
-^^^^^
-
-Assuming you have Schema Registry running, here are the recommended steps to add Schema Registry instances in a new "slave" datacenter (call it DC B):
-
-- In DC B, make sure Kafka has ``unclean.leader.election.enable`` set to false.
-
-- In Kafka in DC B, create the ``_schemas`` topic. It should have 1 partition, ``kafkastore.topic.replication.factor`` of 3, and ``min.insync.replicas`` at least 2.
-
-- In DC B, run Replicator with Kafka in the "master" datacenter (DC A) as the source and Kafka in DC B as the target.
-
-- In the Schema Registry config files in DC B, set ``kafkastore.connection.url`` and ``schema.registry.zk.namespace`` to match the instances already running, and set ``master.eligibility`` to false.
-
-- Start your new Schema Registry instances with these configs.
-
-Run Book
-^^^^^^^^
-
-Let's say you have Schema Registry running in multiple datacenters, and you have lost your "master" datacenter; what do you do? First, note that the remaining Schema Registry instances will continue to be able to serve any request which does not result in a write to Kafka. This includes GET requests on existing ids and POST requests on schemas already in the registry.
-
-- If possible, revive the "master" datacenter by starting Kafka and Schema Registry as before.
-
-- If you must designate a new datacenter (call it DC B) as "master", update the Schema Registry config files so that ``kafkastore.connection.url`` points to the local ZooKeeper, and change ``master.eligibility`` to true. The restart your Schema Registry instances with these new configs in a rolling fashion.
+If you want to keep things simple, you can take a temporary downtime for Schema Registry and do
+the migration. To do so, simply shutdown all the nodes and start them again with the new configs.
```

Comment:
not sure if this is clear enough re: shut everything down, then restart vs rolling bounce